csfieldguide/templates/chapters/index.html

```diff
@@ -13,15 +13,25 @@ <h1 id="chapters" class="section-heading anchor-link">Chapters</h1>
 
 {% block content %}
 
-  {% if all_chapters %}
-      {% for chapter in all_chapters %}
-        <a href="{% url 'chapters:chapter' chapter.slug %}">
-          <img src="{% if chapter.icon %}{% static chapter.icon %}{% else %}http://placehold.it/238x200{% endif %}">
-          <h3>{{ chapter.name }}</h3>
-        </a>
-      {% endfor %}
-  {% else %}
-    <p>No chapters are available.</p>
-  {% endif %}
+  <div class="row chapter-list">
+    
+      {% if all_chapters %}
+          {% for chapter in all_chapters %}
+          <div class="l12 chapter-list-item">
```

Comment:
`chapter-list-item` has no CSS rules attached to it so should be deleted.

Reply:
see lines 13-15 in `style.css`

Reply:
That could be simplified as:

```
 .chapter-list p {
  color: #000;
}
```